packages/common/.gitignore

```diff
@@ -7,7 +7,5 @@ tmp
 config/authorized_keys
 config/cloudformation.yml
 cache-invalidation/config/runtime_config.json
-/log.js
-/util.js
+/*.js
 /*.d.ts
-/*.d.ts.map
```

Comment:
We don't want to ignore the mapping files now?

Reply:
They're being ignored from here: https://github.com/nasa/cumulus/blob/d5e5ac68770208c1f0e1a6bc43ec0e9466a4fb2b/.gitignore#L18